content/4-component-composition/5-context/vue3/UserProfile.vue

```diff
@@ -0,0 +1,15 @@
+<script setup>
+import { inject } from "vue";
+const { user, updateUsername } = inject("userContextProvider");
+</script>
+
+<template>
+  <div>
+    <h2>My Profile</h2>
+    <p>Username: {{ user.username }}</p>
+    <p>Email: {{ user.email }}</p>
+    <button @click="updateUsername('Jane')">
```

Comment:
You are calling the function directly, that's wrong. It must be a function declaration, not a function call.